src/NuGet.Core/NuGet.Protocol/Plugins/EmbeddedSignatureVerifier.cs

```diff
@@ -32,7 +32,7 @@ public static EmbeddedSignatureVerifier Create()
                 return new WindowsEmbeddedSignatureVerifier();
             }
 
-            if (RuntimeEnvironmentHelper.IsLinux || RuntimeEnvironmentHelper.IsMacOSX || RuntimeEnvironmentHelper.IsMono)
+            if (RuntimeEnvironmentHelper.IsFreeBSD || RuntimeEnvironmentHelper.IsLinux || RuntimeEnvironmentHelper.IsMacOSX || RuntimeEnvironmentHelper.IsMono)
```

Comment:
```suggestion
            if (RuntimeEnvironmentHelper.IsLinux || RuntimeEnvironmentHelper.IsMacOSX || RuntimeEnvironmentHelper.IsFreeBSD || RuntimeEnvironmentHelper.IsMono)
```

Since C# does lazy evaluation, in order, it's better to put the most likely expression first, least likely expression last, to minimize work.

Reply:
Since I removed the `RuntimeEnvironmentHelper.IsFreeBSD` property again, this is no longer relevant.